build/Helix/runtests.cmd

```diff
@@ -1,6 +1,13 @@
 robocopy %HELIX_CORRELATION_PAYLOAD% . /s /NP
 
-te MUXControls.Test.dll MUXControlsTestApp.appx IXMPTestApp.appx /enablewttlogging /unicodeOutput:false /sessionTimeout:0:15 /testtimeout:0:10 /screenCaptureOnError %* 
+reg add HKLM\Software\Policies\Microsoft\Windows\Appx /v AllowAllTrustedApps /t REG_DWORD /d 1 /f
+
+cd scripts
+powershell -Command Add-AppxPackage ../Microsoft.VCLibs.%testbuildplatform%.14.00.appx
+powershell -Command Add-AppxPackage ../Microsoft.NET.CoreRuntime.1.1.appx
+cd ..
+
+te MUXControls.Test.dll MUXControlsTestApp.appx IXMPTestApp.appx /enablewttlogging /unicodeOutput:false /sessionTimeout:0:15 /testtimeout:0:10 /screenCaptureOnError /APPX:Unregister=false %* 
```

Comment:
is /appx:unregister a param to our code or to TE itself? What does it do?

Reply:
It is a param to TE. It is used in the case where you run against an appx. This flag controls whether te will uninstall the appx once the test completes.

Reply:
Yeah, shouldn't be needed anymore. It helped with the pass rate before the add-appxpackage calls.